[2023-10-20 23:21:57,102] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-02-13T00:00:00+00:00 [queued]>
[2023-10-20 23:21:57,119] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-02-13T00:00:00+00:00 [queued]>
[2023-10-20 23:21:57,120] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:21:57,121] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-10-20 23:21:57,122] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:21:57,150] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2023-02-13 00:00:00+00:00
[2023-10-20 23:21:57,158] {standard_task_runner.py:52} INFO - Started process 2032 to run task
[2023-10-20 23:21:57,163] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2023-02-13T00:00:00+00:00', '--job-id', '593', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpyqwcucm8', '--error-file', '/tmp/tmp72ty5cvx']
[2023-10-20 23:21:57,165] {standard_task_runner.py:80} INFO - Job 593: Subtask segunda_tarea
[2023-10-20 23:21:57,277] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-02-13T00:00:00+00:00 [running]> on host dfdda228c9fc
[2023-10-20 23:21:57,396] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2023-02-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-13T00:00:00+00:00
[2023-10-20 23:21:57,398] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-10-20 23:21:57,400] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-10-20 23:21:57,411] {subprocess.py:85} INFO - Output:
[2023-10-20 23:21:57,413] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-10-20 23:21:57,414] {subprocess.py:96} INFO - Command exited with return code 0
[2023-10-20 23:21:57,454] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20230213T000000, start_date=20231020T232157, end_date=20231020T232157
[2023-10-20 23:21:57,497] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-20 23:21:57,567] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
